app/src/main/java/com/eveningoutpost/dexdrip/watch/miband/MiBandService.java

```diff
@@ -91,69 +102,45 @@
  */
 
 public class MiBandService extends JamBaseBluetoothSequencer {
-    private static final boolean d = true;
-
+    public static final boolean d = true;
```

Comment:
Should be a more readable variable name.
Is this variable really necessary? It never changes to false and hence the debug messages will be logged, always.
Similar things can be achieved using verbose logging.